tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -190,6 +190,7 @@ dependency,dependencyUrl,licenseOverride,copyright,sourceURL
 "snappy-jars:",https://github.com/doxavore/snappy-jars,Apache-2.0
 "snappy:",https://github.com/miyucy/snappy,MIT
 "snmp:",https://github.com/hallidave/ruby-snmp,MIT
+"strscan":https://github.com/ruby/strscan,BSD-2-Clause
```

Comment:
This csv line is malformed, needs to include the : inside the dependency name. 

```suggestion
"strscan:",https://github.com/ruby/strscan,BSD-2-Clause
```

The : is there in case there's a need to specify a version.